voronoi.cpp

```diff
@@ -36,7 +36,7 @@ Variant VoronoiSite::edges() const {
 	Vector<Variant> result;
 	const jcv_graphedge* graphedge = _site->edges;
 	while (graphedge) {
-		result.push_back(_diagram->_edges_by_address[Variant((long)graphedge->edge)]);
+		result.push_back(_diagram->_edges_by_address[Variant((uint64_t)graphedge->edge)]);
```

Comment:
I actually pushed some more commits to master from myself where I fix some compiler warnings on MSVC, including the plain conversion from pointers to long. In 6b9b1b7ab9b01e377098c1cfe62df555d3f30038, I replaced this C style cast with something that is supposed to be platform-neutral. Could you maybe pull the master branch into your master branch and check if my commit also works fine on Android? :)